apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -0,0 +1,89 @@
+/*global ServiceMixin */
+'use strict';
+
+importScripts('../service_mixin.js');
+
+(function() {
+  var service;
+
+  var ConversationService = {
```

Comment:
Should we also at least define the list of events that service will broadcast in the scope of this bug, or you think we should add it in the follow ups?